docs/release-notes/0.9.2.md

```diff
@@ -1,4 +1,5 @@
 ### 0.9.2 {small}`the future`
 
 ```{rubric} Bugfix
+* Fix ufuncs of views like ``adata.X[:10].cov(axis=0) returning views {pr}`1043` {user}`flying-sheep`
```

Comment:
Look at the other release notes for usage

Reply:
???

Reply:
Text doesn't go inside the ticks:

https://raw.githubusercontent.com/scverse/anndata/main/docs/release-notes/0.8.1.md